### PR TITLE
Chore: Fix dependencies conflict

### PR DESCRIPTION
**Pull request by renancaraujo**
# Description

After trying to run `flutter pub get` on flame I got some constraint-related conflicts, this PR fixes that.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Comment by wolfenrain** [comment]
Looks like your PR also broke our analyze job :-(

File: packages/flame/analysis_options.yaml [hunk 1]
```diff
@@ -134,15 +134,15 @@ linter:
 
 dart_code_metrics:
   rules:
-    # - prefer-trailing-comma
+    - prefer-trailing-comma
```
Comment (luanpotter) [inline packages/flame/analysis_options.yaml:137]: 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Review by luanpotter — APPROVED** [review]
LGTM

**Review by erickzanardo — APPROVED** [review]
LGTM

**Review by spydon — APPROVED** [review]
LGTM!

**Pull request merged**